src/main/java/pl/edu/agh/firecell/model/material/MaterialConductionMap.java

```diff
@@ -0,0 +1,59 @@
+package pl.edu.agh.firecell.model.material;
+
+import pl.edu.agh.firecell.model.exception.ConductionCoefficientException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MaterialConductionMap {
+
+    private final Map<Pair, Double> coefficientMap;
+
+    public MaterialConductionMap() {
+        coefficientMap = new HashMap<>();
+    //==================================================================================================================
+        coefficientMap.put(new Pair(Material.AIR, Material.AIR),                                            /**/ 0.008);
+        coefficientMap.put(new Pair(Material.AIR, Material.WOOD),                                            /**/ 0.02);
+        coefficientMap.put(new Pair(Material.AIR, Material.CELLULAR_CONCRETE),                              /**/ 0.015);
+    //==================================================================================================================
+        coefficientMap.put(new Pair(Material.WOOD, Material.WOOD),                                            /**/ 0.2);
+        coefficientMap.put(new Pair(Material.WOOD, Material.CELLULAR_CONCRETE),                              /**/ 0.15);
+    //==================================================================================================================
+        coefficientMap.put(new Pair(Material.CELLULAR_CONCRETE, Material.CELLULAR_CONCRETE),                  /**/ 0.1);
+    //==================================================================================================================
+        try {
+            validate();
+        } catch (ConductionCoefficientException e) {
+            e.printStackTrace();
```

Comment:
The error such as having this map wrong is rather demolishing, and you should not handle it here, especially by doing nothing XDD
Just let this error propagate, it's very important
We do it all the time, the place where error occured is nearly never the place where we should deal with it. Just propagate (by adding `throws` declaration to this class
And spoiler: we can do absolutely nothing with this error, because its so powerful. Just let it propagate and not be catched. Im not sure, but we can achieve that by not using `try catch` where you use it, just skip it, and this error should immediately crush an app, which is desired behavior.